coalib/bearlib/aspects/Spelling.py

```diff
@@ -42,6 +42,30 @@ class docs:
 
 
 @Spelling.subaspect
+class CompanySpecificWordSpelling:
+    """
+    This aspect checks for companies' or organisations' (like coala) specific
+    words' spelling.
+    """
+    class docs:
+        example = """
+        `Coala`, `Aspects`, `aspectYEAH`.
```

Comment:
maybe change the example to `Coala is always written with a lower case c, also at the beginning of the sentence.` then it's a self explaining example